tools/ci_build/github/azure-pipelines/templates/py-packaging-stage.yml

```diff
@@ -132,7 +132,7 @@ stages:
               -e NIGHTLY_BUILD \
               -e BUILD_BUILDNUMBER \
               onnxruntimecpubuild \
-                bash /onnxruntime_src/tools/doc/builddoc.sh $(PythonManylinuxDir)/bin/ /onnxruntime_src /build Release
+                bash -c " $(PythonManylinuxDir)/bin/python3 -m pip install /build/Release/dist/onnxruntime*.whl && /onnxruntime_src/tools/doc/builddoc.sh $(PythonManylinuxDir)/bin/ /onnxruntime_src /build Release " ;
```

Comment:
If it is a nightly build, then the package name is ort_nightly or ort_gpu_nightly. onnxruntime*.whl doesn't match any.